### DISCUSSION TITLE
Font color icon not showing

### DISCUSSION THREAD

**Discussion by Colartegi** [webfashionist/RichText#110] [Q&A] [answered]
Hi. 
Does not show font color icon. 
![Sin título](https://github.com/webfashionist/RichText/assets/61756935/8dd01bd0-cea3-4765-a68a-ce7da4c54f19)
Even if I use the other option icon instead, it doesn't show it.
I'm using RichText version 1.1.0, jQuery v3.6.4,Font Awesome Free 5.0.9

**Comment by webfashionist** [comment, marked as answer]
Hi, thank you for letting me know. 

I have added a fix with [ac982b2](https://github.com/webfashionist/RichText/commit/ac982b242c372274e81417bed9699a0d6ae60831)
